images/cilium/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1.1-experimental
+
+# Copyright 2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG CILIUM_BUILDER_IMAGE=docker.io/cilium/cilium-builder-dev:a116c9e46733fce354b871dd6e88e037d9459286
+ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:bf5a13ab0dd341b446ec1f40a809d365024eeb09
+
+FROM --platform=linux/amd64 ${CILIUM_BUILDER_IMAGE} as builder
```

Comment:
Please leave a comment to what this `--platform` is doing.

Reply:
I've documented a simplified version of this pattern in the README, since it applied to other Dockerfiles also.